cmd/opentelemetry/app/receiver/jaegerreceiver/jaeger_receiver.go

```diff
@@ -111,6 +112,30 @@ func configureCollector(v *viper.Viper, cfg *jaegerreceiver.Config) {
 			Endpoint: cOpts.CollectorHTTPHostPort,
 		}
 	}
+
+	// if at least one option is set, we assume they are all provided and let
+	// the underlying mechanism to handle missing values
+	if cfg.GRPC != nil && len(cOpts.AuthOIDCIssuerURL) > 0 {
+		// TODO: how can we log a warning if the user fails to specify a conditionally required flag?
+		// if authOIDCIssuerURL is set, authOIDCClientID has to be set as well (and vice-versa).
+		cfg.GRPC.Auth = &configauth.Authentication{
```

Comment:
Unsure if it's possible atm, but I feel like an unsigned token mode would be valuable.  If you didn't want to muck with oidc but just wanted to generate some simple tokens for multitenancy.

Reply:
Let's record this as an issue. I don't think people would actually do that in production environments, but I could be wrong.